tests/test_filters.py

```diff
@@ -10,7 +10,7 @@
 """
 import random
 import pytest
-from jinja2 import Markup, Environment
+from jinja2 import Markup, Environment, Template
```

Comment:
Any reason to use `Template(str)` instead of `env.from_string(str)`?

Reply:
env.from_string(str) didn't check the regex. So if the user were to use an older version of Python, `env.from_string()` was failing.